src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -18,7 +20,7 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
 
     const swaNode: StaticWebAppTreeItem = await node.createChild(context);
 
-    const createdSs: string = localize('createdSs', 'Successfully created new static web app "{0}".', swaNode.name);
+    const createdSs: string = localize('createdSs', 'Successfully created new static web app "{0}".  The GitHub Action must complete before your Static Web App will receive the deployed content.', swaNode.name);
```

Comment:
"GitHub Actions is buildig and deploying your app, it will be available once the deployment completes."